src/runtime/virtcontainers/clh.go

```diff
@@ -287,12 +287,14 @@ func (clh *cloudHypervisor) createSandbox(ctx context.Context, id string, networ
 
 	// set the serial console to the cloud hypervisor
 	if clh.config.Debug {
-		clh.vmconfig.Serial = chclient.NewConsoleConfig(cctTTY)
+		clh.vmconfig.Console = chclient.NewConsoleConfig(cctTTY)
+		clh.vmconfig.Console.SetMode("Tty")
+		clh.vmconfig.Console.SetIommu(true)
```

Comment:
Why are you setting `iommu=on` here? This is gonna put the device behing a virtual IOMMU, which I don't really understand why you'd need that.

Reply:
Yep - both those lines can go - updated.